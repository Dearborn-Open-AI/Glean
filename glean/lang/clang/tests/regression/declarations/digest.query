query: digest.FileDigest _
transform: [gensort, []]
