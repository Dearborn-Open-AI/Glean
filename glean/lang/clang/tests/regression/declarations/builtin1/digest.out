[
  "@generated",
  {
    "key": {
      "file": { "key": "test.cpp" },
      "digest": {
        "hash": "05d9287d4717a2a6eb1e5a1c264ee9976e497ee5",
        "size": 316
      }
    }
  }
]